apps/extension/src/wasm-task.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Action,
+  TransactionPlan,
+  WitnessData,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
+import { JsonValue } from '@bufbuild/protobuf';
+
+interface WasmTaskInput {
+  transactionPlan: JsonValue;
+  witness: JsonValue;
+  fullViewingKey: string;
+  actionId: number;
+}
```

Comment:
This should be a shared type between the requestor and handler. So this probably needs exported somewhere?

Reply:
L moved this to the `internal-msg/offscreen.ts` in the `/types` package